bspacks/README.md

```diff
@@ -93,8 +92,6 @@ You can then run:
     -I "$reasonTargetDir" \
     -I "$reasonTargetDir/_build/src" \
     -I "$reasonTargetDir/vendor/cmdliner" \
-    -I "$reasonParserTargetDir/_build/src" \
-    -I "$reasonParserTargetDir/vendor/easy_format" \
```

Comment:
This needs to be added back as `"$reasonTargetDir/vendor/easy_format"`, otherwise it's not included.